### PR TITLE
chore(radar_crossing_objects_noise_filter): fix README

### PR DESCRIPTION
**Pull request by scepter914**
## Description

Fix README of `radar_crossing_objects_noise_filter`

## Tests performed

No need

## Effects on system behavior

No

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by HansRobo** [comment]
I updated this branch to import https://github.com/autowarefoundation/autoware.universe/pull/6278.
This will help you pass build-and-test-differential faster.

File: perception/radar_crossing_objects_noise_filter/README.md [hunk 1]
```diff
@@ -51,22 +51,30 @@ To filter the objects crossing to ego vehicle, this package filter the objects a
...
-| :------------------- | :----- | :-------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
-| `angle_threshold`    | double | The angle threshold parameter to filter [rad]. This parameter has condition that 0 < `angle_threshold` < pi / 2. See algorithm chapter for details. | 1.0472        |
-| `velocity_threshold` | double | The velocity threshold parameter to filter [m/s]. See algorithm chapter for details.                                                                | 3.0           |
+### Parameters
+
+- `angle_threshold` (double) [rad]
+  - Default parameter is 1.0472.
+
+This parameter is the angle threshold to filter. It has condition that 0 < `angle_threshold` < pi / 2. If the crossing angle is larger than this parameter, it can be a candidate for noise object. In other words, if it is smaller than this parameter, it is a filtered object.
+If this parameter is set smaller, more objects are considered noise. In detail. see algorithm chapter.
```
Comment (YoshiRi) [inline perception/radar_crossing_objects_noise_filter/README.md:74]:
```suggestion
If this parameter is set smaller, more objects are considered noise. In detail, see algorithm chapter.
```

File: perception/radar_crossing_objects_noise_filter/README.md [hunk 2]
```diff
@@ -51,22 +51,30 @@ To filter the objects crossing to ego vehicle, this package filter the objects a
...
+
+- `velocity_threshold` (double) [m/s]
+  - Default parameter is 3.0.
+
+This parameter is the velocity threshold to filter. If velocity of an object is larger than this parameter, it can be a candidate for noise object. In other words, if velocity of an object is smaller than this parameter, it is a filtered object.
+If this parameter is set smaller, more objects are considered noise. In detail. see algorithm chapter.
```
Comment (YoshiRi) [inline perception/radar_crossing_objects_noise_filter/README.md:80]:
```suggestion
If this parameter is set smaller, more objects are considered noise. In detail, see algorithm chapter.
```

**Review by YoshiRi — CHANGES_REQUESTED** [review]

**Review by miursh — APPROVED** [review]
LGTM

**Review by YoshiRi — APPROVED** [review]
LGTM

**Pull request merged**